satisfy/linux.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+apt install -y build-essential
+apt install -y libssl-dev
+apt install -y gradle
+apt install -y groovy
+apt install -y nodejs
+apt install -y redis
+apt install -y mysql-server-5.7
+
+# install nvm
+mkdir -p $HOME/.nvm
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+source ~/.bashrc
+
+# Install yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```

Comment:
There's a chance that the yarn from apt is too old for the deck codebase

Was why I switched the MacOS script away from brew install yarn to npm -g install yarn

But, if the UI doesn't complain then sticking with the apt is probably best

Reply:
That's a good point. Since we still have the `npm -g install yarn` later, I think it should be safe to remove this part?

Reply:
At least for now it looks like apt has the latest version, so I'm going to leave it in there.